src/test/java/org/takes/rq/RqMethodTest.java

```diff
@@ -47,4 +49,54 @@ public void returnsMethod() throws IOException {
             Matchers.equalTo(RqMethod.POST)
         );
     }
+
+    /**
+     * RqMethod supports all HTTP methods.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
+    public void supportsAllMethods() throws IOException {
+        for (final String method
+            : Arrays.asList(
+                RqMethod.DELETE, RqMethod.GET, RqMethod.HEAD, RqMethod.OPTIONS,
+                RqMethod.PATCH, RqMethod.POST, RqMethod.PUT
```

Comment:
@bdragan there is method `TRACE` in [RFC](http://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html)